db/migrate/20160720100703_create_tasks.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTasks < ActiveRecord::Migration[5.0]
+  def change
+    create_table :tasks do |t|
+      t.string :title
+      t.text :description
+      t.date :due_date
+      t.boolean :status
```

Comment:
I would stay with status to avoid new migration. Status was proposed by the team.